internal/csi-addons/rbd/replication_test.go

```diff
@@ -512,6 +512,18 @@ func TestValidateLastSyncInfo(t *testing.T) {
 			},
 			expectedErr: "",
 		},
+		{
+			name: "description with last_snapshot_sync_seconds = 0",
+			//nolint:lll // sample output cannot be split into multiple lines.
+			description: `replaying, {"bytes_per_second":0.0,"bytes_per_snapshot":81920.0,"last_snapshot_sync_seconds":0,
+			"last_snapshot_bytes":81920,"local_snapshot_timestamp":1684675261,"remote_snapshot_timestamp":1684675261,"replay_state":"idle"}`,
```

Comment:
am not sure we can have this case from Ceph, where we will have last_snapshot_bytes but duration is `0`

Reply:
> am not sure we can have this case from Ceph, where we will have last_snapshot_bytes but duration is `0`

I did not focus on other parameters.
We can ignore that parameter for this test case since parameters are not interlinked here, this test case only focuses on duration.